src/test/claimtriebranching_tests.cpp

```diff
@@ -300,6 +312,17 @@ struct ClaimTrieChainFixture{
         num_txs_for_next_block = 0; 
     }
 
+
+    void WriteClearReadClaimTrie()
+    {
+        // this will simulate restart of lbrycrdd by writing the claimtrie to disk,
+        // clearing the-in memory claimtrie, and then reading it the saved claimtrie
+        // from disk
+        pclaimTrie->WriteToDisk();
+        pclaimTrie->clear();
+        pclaimTrie->ReadFromDisk(true);
+    }
```

Comment:
This also doesn't do exactly what's in the comment since the Cache isn't modified/updated/rebuilt.

Reply:
What do you mean here exactly ? You may have misssed where WriteToDisk clears out the cache entries https://github.com/lbryio/lbrycrd/blob/master/src/claimtrie.cpp#L1047 here

Reply:
I was referring to the state of the TrieCache.